fuel-crypto/README.md

```diff
@@ -0,0 +1,14 @@
+# Fuel Crypto
+
+[![build](https://github.com/FuelLabs/fuel-crypto/actions/workflows/ci.yml/badge.svg)](https://github.com/FuelLabs/fuel-crypto/actions/workflows/ci.yml)
```

Comment:
these URLs are outdated

Reply:
Good spot! I've updated these and squashed the work into the previous "Fix-up" commit, to try and keep the post-merge fixup all under one commit (seeing as we don't want to squash this PR).